File: product_code_3.js

const productos = [
    { nombre: "Camiseta NFT", precio: 30, token: "USDT" },
    { nombre: "Wallet personalizada", precio: 120, token: "ETH" },
    { nombre: "Stickers edición especial", precio: 10, token: "USDC" }
];

function mostrarProductos() {
    return productos.map(producto => `${producto.nombre} - ${producto.precio} ${producto.token}`).join('\n');
}

console.log(mostrarProductos());

File: product_code_3.py

productos = [
    {"nombre": "Camiseta NFT", "precio": 30, "token": "USDT"},
    {"nombre": "Wallet personalizada", "precio": 120, "token": "ETH"},
    {"nombre": "Stickers edición especial", "precio": 10, "token": "USDC"}
]

def mostrar_productos():
    return [f"{p['nombre']} - {p['precio']} {p['token']}" for p in productos]

print(mostrar_productos())
